app/views/admin/groups/_form.html.haml

```diff
@@ -0,0 +1,29 @@
+= form_for(group, url: url) do |f|
+  - if group.errors.any?
+    .alert.alert-danger
+      %ul
+        - group.errors.full_messages.each do |message|
+          %li
+            = message
+
+  .form-group
+    = f.label :name
+    = f.text_field :name, class: 'form-control'
+
+  .form-group
+    = f.label :description
+    = f.text_area :description, class: 'form-control'
+
+  - Repo.all.each do |repo|
```

Comment:
Should repos be passed from the controller? I always feel a bit funny about database queries in the view...